src/index.js

```diff
@@ -1,93 +1,24 @@
-const fs = require("fs"),
-    http = require("http"),
-    path = require("path"),
-    methods = require("methods"),
-    express = require("express"),
-    bodyParser = require("body-parser"),
-    session = require("express-session"),
-    cors = require("cors"),
-    passport = require("passport"),
-    errorhandler = require("errorhandler"),
-    mongoose = require("mongoose");
-
-const isProduction = process.env.NODE_ENV === "production";
-
+import express from 'express';
+import { serve, setup } from 'swagger-ui-express';
+import SwaggerSpecs from '../public/api-docs/swagger.json';
 
 const app = express();
+const port = process.env.PORT || 3000;
 
-app.use(cors());
-
-
-app.use(require("morgan")("dev"));
-app.use(bodyParser.urlencoded({ extended: false }));
-app.use(bodyParser.json());
-
-app.use(require("method-override")());
-app.use(express.static(__dirname + "/public"));
-
-app.use(
-    session({
-        secret: "authorshaven",
-        cookie: { maxAge: 60000 },
-        resave: false,
-        saveUninitialized: false
-    })
-);
-
-if (!isProduction) {
-    app.use(errorhandler());
-}
-
-if (isProduction) {
-    mongoose.connect(process.env.MONGODB_URI);
-} else {
-    mongoose.connect("mongodb://localhost/conduit");
-    mongoose.set("debug", true);
-}
-
-require("./models/User");
+const router = express.Router();
 
+router.use('/public/api-docs', serve, setup(SwaggerSpecs));
 
+app.use(router);
 
-app.get('/', (req, res) => {
```

Comment:
I think these codes `line 52-3` shouldn't have deleted, because this is making tests to fail, and is affecting your PR's deployment

Reply:
Alright. Let me fix it. Thanks for the feedback.